meetings/ui/src/main/frontend/.gitignore

```diff
@@ -0,0 +1,25 @@
+.DS_Store
```

Comment:
Could we merge this file with Sakai's gitignore?

Reply:
done